### PR TITLE
Answerpage

### PR DESCRIPTION
**Pull request by Devon-McGrath-github**
created the --
    -Question page
    -Add-Answer page
    -Answers page

### CODE REVIEW

File: client/components/Add-Answer.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
```
Comment (don-smith) [inline client/components/Add-Answer.jsx]: Can we please change the name of this file to `AddAnswer.jsx` (without the hyphen)?

File: client/components/Answers.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    return (
+      <div>
+        <h1>Answers:</h1>
+
+        <ul>
+          {answersData.map((answer) => {
+            if (answer.questionId === currentQuestion.id) {
+              return (
+                <li key={answer.id}>
+                  {answer.content} {' '}
```
Comment (don-smith) [inline client/components/Answers.jsx]: What purpose is `{' '}` serving on this line?

File: client/components/App.jsx [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import React from 'react'
+import Question from './Question'
 
 export default React.createClass({
   render () {
     return (
       <div>
-        <h1>Things</h1>
+        <Question />
```
Comment (don-smith) [inline client/components/App.jsx]: I think it still makes sense to have some header text. "Questions"?

File: client/components/Add-Answer.jsx (hunk 1 above)
Comment (don-smith) [inline client/components/Add-Answer.jsx]: I think you need we still need to remove this file from the repo.

Comment (Devon-McGrath-github) [inline client/components/Add-Answer.jsx]: done

File: client/components/Answers.jsx (hunk 2 above)
Comment (Devon-McGrath-github) [inline client/components/Answers.jsx]: done

File: client/components/App.jsx (hunk 3 above)
Comment (Devon-McGrath-github) [inline client/components/App.jsx]: done

**Pull request merged**